src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -545,35 +545,35 @@ namespace QgsWms
   {
     const QMetaEnum metaEnum( QMetaEnum::fromType<ParameterName>() );
 
-    log( "WMS Request parameters:" );
+    log( QStringLiteral( "WMS Request parameters:" ) );
     for ( auto parameter : mParameters.toStdMap() )
     {
       const QString value = parameter.second.mValue.toString();
 
       if ( ! value.isEmpty() )
       {
         const QString name = metaEnum.valueToKey( parameter.first );
-        log( " - " + name + " : " + value );
+        log( QStringLiteral( " - " ) + name + QStringLiteral( " : " ) + value );
```

Comment:
Thank you for updating this file with `QStringLiteral` :+1: 

However something like that could be used with args:

``` c++
QStringLiteral( " - %1 ...." ).arg( name );
```

What do you think?

Reply:
I think it's a lot of work...